tools/prometheus/prometheus.yml

```diff
@@ -1,6 +1,6 @@
 global:
-  scrape_interval: 10s
-  scrape_timeout: 10s
+  scrape_interval: 5s
+  scrape_timeout: 5s
```

Comment:
this configuration is only used in local containers so it is okay to increase the frequency at which local prometheus "scrapes" for metrics from our local otel-collector container.

In live envs, this is outside of our control. OpenShift has a Prometheus operator installed that handles it automatically based on container metadata that we already provide